mkdocs.yml

```diff
@@ -99,4 +99,9 @@ nav:
   #   - repository-location.md
   # - Reference:
   #   - copyright.md
-  #   - trademark-policy.md
\ No newline at end of file
+  #   - trademark-policy.md
```

Comment:
Correct the syntax in the file. See https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/?h=analytics#google-analytics